src/types/clipboard.ts

```diff
@@ -2,6 +2,13 @@ import { CommandResult } from "./commands";
 import { Dimension, HeaderIndex, UID, Zone } from "./misc";
 import { GridRenderingContext } from "./rendering";
 
+export enum ClipboardContentType {
```

Comment:
what about MIMEType ? That's what it is after all
```suggestion
export enum MIMEType {
```

Reply:
`ClipboardMIMEType` then, because I want the enum to be lilited to supported MIME types for the clipboard, not any MIME type.